includes/class-wc-geolocation.php

```diff
@@ -194,6 +192,27 @@ public static function update_database() {
 		WC_Integration_MaxMind_Geolocation::update_database();
 	}
 
+	/**
+	 * Fetches the country code from the request headers, if one is available.
+	 *
+	 * @return string|false The country code pulled from the headers, or false if one was not found.
+	 */
+	private static function get_country_code_from_headers() {
+		$country_code = false;
+
+		if ( ! empty( $_SERVER['HTTP_CF_IPCOUNTRY'] ) ) { // WPCS: input var ok, CSRF ok.
```

Comment:
I think that will be nice to you add _MM_COUNTRY_CODE_ which is often default environment variable for [mod_maxminddb](https://github.com/maxmind/mod_maxminddb).

Maybe it's possible to use _foreach_ instead of few _if conditionals_...  (I don't know is it fine by your coding standards). Something like this:

```php
$var_names = ( 'HTTP_CF_IPCOUNTRY', 'MM_COUNTRY_CODE', 'GEOIP_COUNTRY_CODE', 'HTTP_X_COUNTRY_CODE' );

foreach ( $var_names as $var ) {
	if ( ! empty( $_SERVER[ $var ] ) ) {
		....
		break;
	}
}
```

Reply:
Thank you for this suggestion, I've gone ahead and made a change.